src/Sarif/FileSearcherHelper.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    internal static class FileSearcherHelper
+    {
+        /// <summary>
+        /// This method will search in the environment variable for a specific file name.
+        /// It will return the first file found.
+        /// </summary>
+        /// <param name="environmentVariable">Environment variable that we will look for</param>
+        /// <param name="fileName">Name of the file that we will look for in the environment variable</param>
+        /// <returns>Path to the file name or empty string.</returns>
+        public static string SearchForFileInEnvironmentVariable(string environmentVariable, string fileName)
+        {
+            string variable = Environment.GetEnvironmentVariable(environmentVariable);
+            if (string.IsNullOrEmpty(variable))
+            {
+                return string.Empty;
```

Comment:
>string.Empty [](start = 23, length = 12)

let's return null. it's a value that's more likely to result in a crash if someone is mishandling the negative condition.